x-pack/plugins/observability_solution/observability/public/components/alert_search_bar/containers/use_alert_search_bar_state_container.tsx

```diff
@@ -24,7 +24,7 @@ import { useTimefilterService } from '../../../hooks/use_timefilter_service';
 
 import {
   useContainer,
-  defaultState,
+  defaultState as DEFAULT_STATE,
```

Comment:
should we rename the defaultState constant to DEFAULT_STATE in ./state_container instead of using an alias in a few different places?